src/Jackett.Common/Indexers/Definitions/Abstract/PublicBrazilianIndexerBase.cs

```diff
@@ -0,0 +1,387 @@
+                _ => ExtractResolution(defaultTitle)
+            };
+            var title = (defaultTitle, description?.TextContent) switch
+            {
+                (string defTitle, _) when !string.IsNullOrWhiteSpace(defTitle) => CleanTitle(defTitle),
+                (_, string text) when !string.IsNullOrWhiteSpace(text) => CleanTitle(text),
+                _ => defaultTitle
+            };
+            return FormatTitle(title, resolution);
+        }
+
+        private string ExtractResolution(string text)
+        {
+            var resolution = "";
+            RowParsingExtensions.ExtractPattern(text, @"\b(\d{3,4}p)\b", res => resolution = res);
+            return resolution;
+        }
+
+        private string FormatTitle(string title, string resolution = null)
+        {
+            return string.IsNullOrWhiteSpace(resolution)
+                ? $"{title}"
+                : $"{title} {resolution}";
+        }
+
+        public long ExtractSizeByResolution(string title)
+        {
+            var resolution = "Other";
+            RowParsingExtensions.ExtractPattern(
+                title, @"\b(\d{3,4}p)\b", res =>
+                {
+                    resolution = res;
+                });
+
+            var size = resolution switch
+            {
+                "720p" => "1GB",
+                "1080p" => "2.5GB",
+                "2160p" => "5GB",
+                _ => "512MB"
+            };
+
+            return RowParsingExtensions.GetBytes(size);
+        }
+
+        protected static string CleanTitle(string title)
+        {
+            if (string.IsNullOrWhiteSpace(title))
+                return null;
+
+            // Remove size info in parentheses
+            title = Regex.Replace(title, @"\(\d+(?:\.\d+)?\s*(?:GB|MB)\)", "", RegexOptions.IgnoreCase);
+
+            // Remove quality info
+            title = Regex.Replace(title, @"\b(?:720p|1080p|2160p|4K)\b", "", RegexOptions.IgnoreCase);
+
+            // Remove source info
+            title = Regex.Replace(title, @"\b(?:WEB-DL|BRRip|HDRip|WEBRip|BluRay|Torrent|Download)\b", "", RegexOptions.IgnoreCase);
+
+            // Remove language info
+            title = Regex.Replace(title, @"\b(?:Legendado|Leg|Dublado|Dub|[AÁ]udio)\b", "", RegexOptions.IgnoreCase);
+
+            // Remove brackets/parentheses content
+            title = Regex.Replace(title, @"\[(?:.*?)\]|\((?:.*?)\)", "", RegexOptions.IgnoreCase);
+
+            // Remove dangling punctuation and separators
+            title = Regex.Replace(title, @"[\\/,|~_-]+\s*|\s*[\\/,|~_-]+", " ", RegexOptions.IgnoreCase);
+
+            // Clean up multiple spaces
+            title = Regex.Replace(title, @"\s+", " ");
+
+            // Clean up torrent group names
+            title = Regex.Replace(title, @"HIDRATORRENTS\.ORG|\[?Erai-raws\]?|\[?Anime Time\]?|COMANDO4K\.COM|COMANDO\.TO", "", RegexOptions.IgnoreCase);
+
+            // Remove file extension from the beginning of title
```

Comment:
the removal of `HIDRATORRENTS.ORG*MKV}MP4` for redetorrent and apachetorrent does not appear to be working
![image](https://github.com/user-attachments/assets/9bfc45cb-05f7-4a76-b466-fcc331b6dfa1)
![image](https://github.com/user-attachments/assets/f8cd0ebe-97fb-4ab4-9e37-67a6a4bd6c9b)

Reply:
Fixed. And added two more groups VACATORRENT and VEMTORRENT